singer_sdk/connectors/sql.py

```diff
@@ -798,12 +798,42 @@ def prepare_table(
             )
             return
 
+        self.prepare_table_columns(
+            full_table_name=full_table_name,
+            schema=schema,
+        )
+
+    def prepare_table_columns(
```

Comment:
I think we should tests for this. `tests/core/test_connector_sql.py` might a good place for that.